diff -up rosegarden-22.06/src/gui/general/ActionData.cpp.omv~ rosegarden-22.06/src/gui/general/ActionData.cpp
--- rosegarden-22.06/src/gui/general/ActionData.cpp.omv~	2022-10-27 01:54:58.944868431 +0200
+++ rosegarden-22.06/src/gui/general/ActionData.cpp	2022-10-27 01:55:53.732348392 +0200
@@ -740,7 +740,8 @@ void ActionData::fillModel()
         m_model->setItem(0, 3, item);
         m_model->item(0, 3)->setEditable(false);
         if (ainfo.global) {
-            QVariant bg(QBrush(Qt::cyan));
+	    QBrush br(Qt::cyan);
+            QVariant bg(br);
             for (int col=0; col<8; col++) {
                 m_model->setData(m_model->index(0, col),
                                  bg, Qt::BackgroundRole);
