diff -up rosegarden-20.06/src/gui/general/ThornStyle.cpp.omv~ rosegarden-20.06/src/gui/general/ThornStyle.cpp
--- rosegarden-20.06/src/gui/general/ThornStyle.cpp.omv~	2020-06-11 23:22:25.115090466 +0200
+++ rosegarden-20.06/src/gui/general/ThornStyle.cpp	2020-06-11 23:22:32.981173588 +0200
@@ -34,6 +34,7 @@
 #include <QLabel>
 #include <QLayout>
 #include <QPainter>
+#include <QPainterPath>
 #include <QRadioButton>
 #include <QStyleFactory>
 #include <QStyleOption>
diff -up rosegarden-20.06/src/gui/widgets/Panner.cpp.omv~ rosegarden-20.06/src/gui/widgets/Panner.cpp
--- rosegarden-20.06/src/gui/widgets/Panner.cpp.omv~	2020-06-11 23:21:14.604345384 +0200
+++ rosegarden-20.06/src/gui/widgets/Panner.cpp	2020-06-11 23:21:23.378438097 +0200
@@ -25,6 +25,7 @@
 #include "base/Profiler.h"
 
 #include <QPolygon>
+#include <QPainterPath>
 #include <QMouseEvent>
 
 #include <iostream>
